pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -375,7 +375,8 @@ type ClusterRoleTemplateBinding struct {
 	// +optional
 	GroupPrincipalName string `json:"groupPrincipalName,omitempty" norman:"noupdate,type=reference[principal]"`
 
-	// ClusterName is the name of the cluster to which a subject is added. Immutable.
+	// ClusterName is the name of the cluster (not displayName) to which a subject is added.
+	// Must match the namespace. Immutable.
```

Comment:
While you are here, would you mind updating this to state that this is the `metadata.name` of the cluster? Cluster's also have a `DisplayName` field I think (which is what shows in the UI), but you need to use the `metadata.name` or it won't work (and I think they are often different).